service/common/util/Message.kt

```diff
@@ -118,8 +118,8 @@ abstract class Message(codePrefix: String, codeNumber: Int, messagePrefix: Strin
                 Connection(15, "Failed to create database '%s', due to duplicate.")
             val FAILED_TO_LOAD_SCHEMA =
                 Connection(16, "Failed to fetch schema for database '%s', due to:\n%s")
-            val CREDENTIALS_EXPIRE_SOON =
-                Connection(17, "Your credentials are expiring within %s days. Use TypeDB Console to update your password.")
+            val CREDENTIALS_EXPIRE_SOON_HOURS =
```

Comment:
We're representing this as hours, mirroring our implementation in Console:
https://github.com/vaticle/typedb-console/commit/5399043b32bb7941803ebfb12291ca1939eb60d0#diff-d55afa5d2520f24b69f54023a399be5ebf476342147d4d7c4471e5a37baef6e9R470

If we get the number of remaining days using `Duration.toDays()`, the user will see "0 days" for 23 hours and 59 minutes before their password expires, which isn't ideal. This way, they only see "0 hours" for an hour.

Reply:
Agreed, done! https://github.com/vaticle/typedb-console/pull/202